build.gradle

```diff
@@ -70,6 +70,14 @@ configurations.all {
         force "org.apache.commons:commons-lang3:3.4"
         force "org.springframework:spring-core:5.3.20"
         force "com.google.guava:guava:30.0-jre"
+        force "com.fasterxml.woodstox:woodstox-core:6.4.0"
+        force "org.scala-lang:scala-library:2.13.9"
+        force "org.yaml:snakeyaml:1.32"
+        force 'io.netty:netty-buffer:4.1.78.Final'
```

Comment:
Do we need to force the resolution of the netty version here?

Reply:
There was a netty cve in the previous push without the netty change. I know that in later versions we have a netty.version variable but this does not seem to exist for 1.x.

Reply:
Yeah, I think I may have seen the CVE for 1.3 on Netty and then because I just copy+pasted the same changes it may have been included. I can remove it if you think I should but I imagine keeping it is not a bad idea either.

Reply:
Alright I can remove snakeyaml and then swap netty to be 79